### PR TITLE
Update documents for v22.02.00

### PR DESCRIPTION
**Pull request by gigony**
Update outdated CHANGELOG.md and README.md files.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,236 +20,9 @@ Please see https://github.com/rapidsai/cucim/releases/tag/v22.04.00a for the lat
 
 ## 🚀 New Features
 
-1. Update cucim.skimage API to match scikit-image 0.19 ([#190](https://github.com/rapidsai/cucim/pull/190)) [@glee77](https://github.com/glee77)
```
Comment (jakirkham) [inline CHANGELOG.md:23]: Should we be deleting this stuff or simply moving it?

Comment (gigony) [reply CHANGELOG.md:23]: The same content (detailed information on new features) is in our release notes so wanted to simplify it in CHANGELOG.md

Comment (grlee77) [reply CHANGELOG.md:23]: okay, that makes sense

**Review by grlee77 — APPROVED** [review]
Thanks, Gigon

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
@gpucibot merge

**Comment by jakirkham** [comment]
Thanks Gigon! 😄

**Pull request merged**